packages/react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -814,6 +814,25 @@ describe('ComboBox', () => {
     });
   });
 
+  it('correctly handles (aria-labelledby) when no label prop is provided', () => {
+    const customId = 'customAriaLabel';
```

Comment:
Are these `customId` consts really necessary? It seems like they don't really affect the outcome of the tests.